.github/workflows/test.yml

```diff
@@ -161,21 +161,35 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
       - name: Setup
         uses: ./.github/actions/setup
+
       - name: TurboRepo local server
         if: ${{ github.event.inputs.turborepo_caching != 'Off' }}
         uses: felixmosh/turborepo-gh-artifacts@v3
         with:
           server-token: 'local'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Test init script
         run:
           pnpm --filter @custom/init run init --project-human-name "Foo Bar"
           --project-machine-name foo_bar | grep 'Run `pnpm i` to update the lock
           file'
+
       - name: Run "pnpm i" to ensure all packages are linked
-        run: pnpm i --no-frozen-lockfile
+        run: |
+          set +e
+          output=$(pnpm i --no-frozen-lockfile 2>&1)
+          exit_code=$?
+          echo "$output"
+          if echo "$output" | grep -q "is not in the npm registry"; then
+            echo "Looks like a new @amazeelabs package was added in this PR. Skipping the package link check."
+            exit 0
```

Comment:
I don't like that, but see no other way around. If we don't `exit 0`, Dev will be always red.